src/Network/Network/AzureFirewallPolicy/NetworkRuleCondition/NewAzureFirewallPolicyNetworkRuleCommand.cs

```diff
@@ -76,7 +76,7 @@ public override void Execute()
                 SourceAddresses = this.SourceAddress?.ToList(),
                 DestinationAddresses = this.DestinationAddress?.ToList(),
                 DestinationPorts = this.DestinationPort?.ToList(),
-                RuleType = "NetworkRuleCondition"
+                RuleType = "NetworkRule"
```

Comment:
are these two rule type equivalent?

Reply:
Yes this is just a name change for consistency.